docs/can-canjs/canjs.md

```diff
@@ -437,6 +437,9 @@ pre[class*=language-].line-numbers.line-numbers code {
   }
 }
 @@media (max-width: 529px) {
+  .hero-logo {
+    width: 260px;
+  }
```

Comment:
Should this not be set on the SVG since the height is set on it?

Reply:
This reduces the width of the logo at the smallest mobile dimension.

Reply:
Is the aspect ratio correct even if the height isn’t set as well?

Reply:
Yes, the aspect ratio is set by the width. The height as been removed.